static/js/index.js

```diff
@@ -0,0 +1,133 @@
+const API_PREFIX = 'api';
+const baseUrl = window.location.origin;
+
+const setScrapingLogs = (logs) => {
+  const logsNode = document.getElementById('scraping-logs');
+  logs.forEach((el) => {
+    const pNode = document.createElement('p');
+    pNode.innerText = el;
+    logsNode.appendChild(pNode);
+  });
+};
+
+const setRadioButtons = (scraperSettings) => {
+  document.querySelectorAll(`[name="loop-radio-group"][value="${scraperSettings.loop_mode.toString()}"]`)[0].checked = true;
```

Comment:
Why do you use querySelectoAll instead of just querySelector? You can skip the [0] part after as well.